app/components/admin/edition_images/image_component.rb

```diff
@@ -45,4 +45,8 @@ def links
     links << link_to("Delete image", confirm_destroy_admin_edition_image_path(@edition, image), class: "govuk-link gem-link--destructive")
     links
   end
+
+  def can_have_lead_image?
+    edition.can_have_custom_lead_image?
+  end
```

Comment:
yea this is a hangover form when it was called multiple times, but it should have been `can_have_custom_lead_image?`. I've removed it and just used `edition.can_have_custom_lead_image?` in the view since it's only called once.